pygmt/src/colorbar.py

```diff
@@ -101,6 +103,18 @@ def colorbar(self, **kwargs):
     {panel}
     {perspective}
     {transparency}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # Create a new plot with pygmt.Figure()
+    >>> fig = pygmt.Figure()
+    >>> # Call the colorbar method for the plot
+    >>> # Set cmap to the "roma" CPT
+    >>> # Label the x-axis "Velocity" and the y-axis "m/s"
+    >>> fig.colorbar(cmap="roma", frame=["x+lVelocity", "y+lm/s"])
+    >>> # Show the plot
+    >>> fig.show()
```

Comment:
Is this inline example already ready for review @willschlitzer?

I think this example does not work, as the `region` and `projection` parameters are not set.

_Modified code examples_
```python
import pygmt

# (I) Set region and projection via pygmt.Figure.basemap
fig = pygmt.Figure()
fig.basemap(region=[0, 10, 0, 10], projection="X10c")
fig.colorbar(cmap="roma", frame=["x+lVelocity", "y+lm/s"])
fig.show()

# (II) Set region and projection via pygmt.Figure.colorbar
fig = pygmt.Figure()
fig.colorbar(
    region=[0, 10, 0, 10],
    projection="X10c",
    cmap="roma",
    frame=["x+lVelocity", "y+lm/s"],
)
fig.show()
```

_Output figures_
(I) Set `region` and `projection` via `pygmt.Figure.basemap`:
![inline_cb_JRviabasemap](https://user-images.githubusercontent.com/94163266/219685810-7d574988-f309-4e30-8aa4-2cd720feffca.png)

(II) Set `region` and `projection` via `pygmt.Figure.colorbar`:
![inline_cb_JRviacb](https://user-images.githubusercontent.com/94163266/219685897-c9c2d52c-3923-42c5-ab14-efa7b7083b25.png)

Reply:
Good catch! I'm fighting with my GMT install and wanted to push to a remote branch before messing with my local machine and forgot to mark this WIP!

Reply:
Actually, I prefer @yvonnefroehlich's first way, because in most cases, people already have their `projection` and `region` defined in previous commands.
```
import pygmt

fig = pygmt.Figure()
fig.basemap(region=[0, 10, 0, 3], projection="X10c/3c")
fig.colorbar(cmap="roma", frame=["x+lVelocity", "y+lm/s"])
fig.show()
```